### ISSUE TITLE
An error occurs when adding communication user to a group

### ISSUE THREAD

**Issue by David-hughes8** [OfficeDev/microsoft-teams-apps-company-communicator#855]
A "something has went wrong" error occurs when we try and add the User TC module to a group. 

The authentication checks out, and everything is in stall albeit via a pipeline not via the PowerShell.

Any ideas of where the failure may be would be appreciated.

Thanks
David Hughes

**Comment by v-royavinash** [comment]
Hi @David-hughes8 , Could you please share more details on the error? It would be great if you share screenshots that would be helpful for our troubleshooting.

Also, can you elaborate on the pipeline deployment that you mentioned?

**Comment by David-hughes8** [comment]
HI V-Royavinash,

I'm having an issue loading the screen shot into the frame so here's a brief description of it: 

A white box with "Something went wrong."  and a cancel button in the bottom RHS corner.

The pipelines are as follows:

1. Build pipeline -  Builds the .NET solution, Publishes the Cmpany Communicator plus the other three functions, creating .zip files of them and placin

2. Release pipeline 1. - runs the arm template to create the resources within the resource group, and then creates the web app.
3. Release pipeline 2 - deploys the app service and the three functions behind it.

Thanks
David

**Comment by gsv022** [comment]
Hi @David-hughes8 ,
Please cross-verify the graph app registration configuration end-end especially API permissions. And please share the deployment logs as well.

**Comment by David-hughes8** [comment]
Hi Sai

***@***.***

The above are the API permissions for Preprod Company Communicator App.

The following is the deployment log for the send function:

***@***.***

David Hughes
Systems Developer and Tenant Admin, (O365, Azure and SP)
Collaboration and Cloud Services
Enterprise Services,
Home Office

Address:  18th Floor Lunar House, Croydon, Surrey, CR0 1XG

Works Mobile no. 07471 142925
Home No. 07766 013362

Email: ***@***.******@***.***>

Future Leave:
22nd August to 30th August 2022 (inc.).
26th Sept. to 30th Sept. 2022 (inc.)

**Comment by gsv022** [comment]
Hi @David-hughes8 ,
Currently we don't support pipeline thing and its kind of customizable. Because we don't have the actual implementation plan. So it will be difficult for us to address the issue. And could you please share the implementation document with us to get to 
 know more about the process of pipeline.
Thank you.

**Comment by David-hughes8** [comment]
Working with MS engineers we were able to identify the cause of the problem.
It wasn't the pipeline deployment that caused the problem it was the age of the Manifest schema.  The code that was downloaded had a manifest version of 1.5, whilst the latest version is 1.12. 
By raising the manifest version and adding a couple of extra  lines to both user and author manifests as follows:

    "supportsFiles": false,
    "isNotificationOnly": true,
    **"supportsCalling": false,
    "supportsVideo": false**
  }],

Resolved the problem
